tests/test_mcp.py

```diff
@@ -38,6 +40,14 @@ async def test_stdio_server():
         assert result.content == snapshot([TextContent(type='text', text='32.0')])
 
 
+async def test_stdio_server_with_cwd():
+    test_dir = Path(__file__).parent
+    server = MCPServerStdio('python', ['mcp_server.py'], cwd=test_dir)
+    async with server:
+        tools = await server.list_tools()
+        assert len(tools) == 1
```

Comment:
Ideally, there would also be a counter case that verifies this fails without specifying the `cwd`, but the MCP SDK seems to just hang indefinitely when the command fails. Have confirmed locally that this test fails when `cwd=None`